src/win32/Watcher.cpp

```diff
@@ -179,7 +223,10 @@ void Watcher::eventCallback(DWORD errorCode) {
         setError("failed resizing buffers for network traffic");
       }
     } else if (errorCode == ERROR_ACCESS_DENIED) {
-      pollDirectoryChanges();
+      // Access denied can happen when we try to delete the watched folder
+      // In windows server the folder is not removed until folder handler is released
```

Comment:
Let's change this to `In Windows Server 2019 and older...`, as I suspect with 2022+ it behaves the same as current Windows 10 (since that's what it's based on).

Reply:
Ok